sopel/modules/ip.py

```diff
@@ -4,12 +4,13 @@
 # Copyright © 2013, Elad Alfassa <elad@fedoraproject.org>
 # Licensed under the Eiffel Forum License 2.
 
-from __future__ import unicode_literals, absolute_import, print_function, division
-
-import pygeoip
+import geoip2.database
 import socket
 import os
-import gzip
+import tarfile
+from sopel.config.types import StaticSection, FilenameAttribute
+from sopel.module import commands, example
+from sopel.logger import get_logger
```

Comment:
Please sort the stdlib imports alphabetically and put `geoip2.database` in a separate group between stdlib and sopel imports. Also, the items in `from` imports should be alphabetized (here, only the `sopel.config.types` ones need changing).

These will soon be enforced by the style checks, but as long as you're touching these lines you might as well bring them into compliance now. One less file for me to touch in the big cleanup I'm working on locally. :smile_cat:

````python
import os
import socket
import tarfile

import geoip2.database

from sopel.config.types import FilenameAttribute, StaticSection
from sopel.module import …  # etc.
````